flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
why?

Reply:
it sure will never be tested. All `org.kde` is built also on aarch64.

Reply:
Alright, let's remove this.